lib/fs.js

```diff
@@ -2145,7 +2145,7 @@ fs.realpath = function realpath(p, options, callback) {
     // dev/ino always return 0 on windows, so skip the check.
     let id;
     if (!isWindows) {
-      var dev = statValues[0/*ino*/].toString(32);
+      var dev = statValues[0/*dev*/].toString(32);
```

Comment:
This change is fine but what about just removing both comments? They are not particularly useful as the variable names already make this obvious. That same change could then be made in another spot in this file where these unnecessary comments exist.

Reply:
@apapirovski you're right! Should I fix it right here or I can fix in a separate pull request?

Reply:
If it's being fixed, I would prefer it in this PR to avoid unnecessary commits. Other collaborators might have a different opinion.